src/tools/hash-text/hash-text.vue

```diff
@@ -44,11 +31,11 @@ const algos = {
   RIPEMD160,
 } as const;
 
+type Algo = keyof typeof algos;
+const list = Object.keys(algos) as Algo[];
+
 const clearText = ref(
-  'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lacus metus blandit dolor lacus natoque ad fusce aliquam velit.',
+  'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lacus metus blandit dolor lacus natoque ad fusce aliquam velit.'
 );
-const algo = ref<keyof typeof algos>('SHA256');
-const hashedText = computed(() => algos[algo.value](clearText.value).toString());
-
-const { copy } = useCopy({ source: hashedText, text: 'Hash copied to the clipboard' });
+const hashedText = (algo: Algo, value: string) => (value ? algos[algo](value).toString() : '');
```

Comment:
Rename `hashedText` to `hashText` since it's a function and no longer a variable